lib/shortcuts/index.js

```diff
@@ -11,5 +11,6 @@ module.exports = {
   ServiceRole: require('./service-role'),
   Queue: require('./queue'),
   S3KinesisFirehose: require('./s3-kinesis-firehose'),
-  hookshot: require('./hookshot')
+  hookshot: require('./hookshot'),
+  SubscriptionFilterLambda: require('./subscription-filter-lambda')
```

Comment:
I think that `LogSubscriptionLambda` might be more informative about what this shortcut exists to implement. Thoughts?

Reply:
totally agree with you there!

Reply:
rolls off the tongue more easily too